package.json

```diff
@@ -37,55 +37,48 @@
     "url": "https://github.com/FortAwesome/angular-fontawesome/issues"
   },
   "homepage": "https://github.com/FortAwesome/angular-fontawesome",
+  "dependencies": {
+    "@angular/animations": "^6.0.6",
```

Comment:
Maybe remove unnecessary dependencies from here: animations, forms, http and router?